facades/PC/build.gradle.kts

```diff
@@ -30,396 +28,370 @@ dateTimeFormat.timeZone = TimeZone.getTimeZone("UTC")
-jar {
+tasks.named<Jar>("jar") {
     manifest {
         //TODO: Maybe later add the engine's version number into here?
-        def manifestClasspath = configurations.runtimeClasspath.collect { it.getName() }.join(" ")
-        attributes("Main-Class": mainClassName)
-        attributes("Class-Path" : manifestClasspath)
-        attributes("Implementation-Title": "Terasology-" + project.name)
-        attributes("Implementation-Version": env.BUILD_NUMBER + ", " + env.GIT_BRANCH + ", " + env.BUILD_ID)
+        attributes["Main-Class"] = mainClassName
+        attributes["Class-Path"] = configurations["runtimeClasspath"].map { it.name }.joinToString(" ")
+        attributes["Implementation-Title"] = "Terasology-" + project.name
+        attributes["Implementation-Version"] = """${env["BUILD_NUMBER"]}, ${env["GIT_BRANCH"]}, ${env["BUILD_ID"]}"""
     }
 }
 
 configurations {
-    // for fetching modules for running a server
-    modules {
-        transitive = false
+    register("modules") {
+        description = "for fetching modules for running a server"
+        isTransitive = false
     }
 }
 
-run {
+tasks.named<JavaExec>("run") {
     description = "Run 'Terasology' to play the game as a standard PC application"
     group = "terasology run"
 
-    args "-homedir", "-noCrashReport"
-    jvmArgs "-Xmx1536m"
-    workingDir rootDir
+    args = listOf("-homedir", "-noCrashReport")
+    jvmArgs = listOf("-Xmx1536m")
+    workingDir = rootDir
 
-    dependsOn rootProject.moduleClasses
+    dependsOn(":moduleClasses")
 }
 
-task game(type:JavaExec) {
+tasks.register<JavaExec>("game") {
     description = "Run 'Terasology' to play the game as a standard PC application"
     group = "terasology run"
 
     // If there are no actual source modules let the user know, just in case ..
-    if (project(':modules').subprojects.size() == 0) {
-        def out = services.get(StyledTextOutputFactory).create("an-ouput")
-        out.withStyle(StyledTextOutput.Style.FailureHeader).println("NOTE: You're running the game from source without any source modules - that may be intentional (got jar modules?) but maybe not. Consider running `groovyw init` or a variant (see `groovyw usage`)")
+    if (project(":modules").subprojects.isEmpty()) {
+        logger.warn("NOTE: You're running the game from source without any source modules - that may be intentional (got jar modules?) but maybe not. Consider running `groovyw init` or a variant (see `groovyw usage`)")
     }
 
     // Dependencies: natives + all modules & the PC facade itself (which will trigger the engine)
-    dependsOn rootProject.extractNatives
-    dependsOn rootProject.moduleClasses
-    dependsOn classes
+    dependsOn(":extractNatives")
+    dependsOn(":moduleClasses")
+    dependsOn("classes")
 
     // Run arguments
     main = mainClassName
     workingDir = rootDir
-    args "-homedir"
-    jvmArgs ["-Xmx1536m"]
+    args = listOf("-homedir")
+    jvmArgs = listOf("-Xmx1536m")
```

Comment:
Would be nice to make this adjustable in the future via some kind of command line argument...